lib/hammer/backend/ets.ex

```diff
@@ -88,21 +88,21 @@ defmodule Hammer.Backend.ETS do
           {:ok, count :: integer}
           | {:error, reason :: any}
   def count_hit(_pid, key, now, increment) do
-    if :ets.member(@ets_table_name, key) do
-      [count, _] =
-        :ets.update_counter(@ets_table_name, key, [
+    [count | _] =
+      :ets.update_counter(
+        @ets_table_name,
+        key,
+        [
           # Increment count field
           {2, increment},
           # Set updated_at to now
           {4, 1, 0, now}
-        ])
+        ],
+        # Insert {key, 0, created_at, updated_at}
+        {key, 0, now, now}
+      )
 
-      {:ok, count}
-    else
```

Comment:
it seems that you are rmoving the check for ets.member. but what happen if it s not? does the update_counter do the insert? or you are going with that if it does not exists then it s updating and creating it behind the scene?

Reply:
👋 @epinault 

https://www.erlang.org/doc/apps/stdlib/ets.html#update_counter/4 handles the missing value by assuming it becomes the `Default` (fourth arg) and then applying the operations to it (third arg).

Reply:
cool , I saw that but just making sure that was intentional